internal/bundle/bundle.yac.go

```diff
@@ -6,19 +6,46 @@
 package bundle
 
 import (
-	"go.mondoo.com/cnquery/explorer"
-	"go.mondoo.com/cnspec/policy"
 	"gopkg.in/yaml.v3"
+	"encoding/json"
+	"errors"
+	"go.mondoo.com/cnquery/explorer"
+    "go.mondoo.com/cnspec/policy"
```

Comment:
This file is auto-generated but should be formatted once

Reply:
fixed